tcn_hpl/models/ptg_module.py

```diff
@@ -367,9 +328,9 @@ def training_step(
             "train/acc", self.train_acc, on_step=False, on_epoch=True, prog_bar=True
         )
 
-        self.log(
-            "train/lr", self.lr_schedulers().get_last_lr()[0], on_step=False, on_epoch=True, prog_bar=True
-        )
+        # self.log(
+        #     "train/lr", self.lr_schedulers().state_dict()[[0], on_step=False, on_epoch=True, prog_bar=True
+        # )
```

Comment:
Dead code.

Reply:
I kept it for @cfunk1210 who runs lighting on a different version and is able to call and log the last_lr() function.